packages/corvid-cli/src/electron/open-editor.js

```diff
@@ -16,7 +16,7 @@ app.on("ready", async () => {
     );
   }
 
-  await openWindow(openEditorApp(), {
+  await openLocalEditorAndServer(openEditorApp(), {
     show: true && !process.env.CORVID_FORCE_HEADLESS
```

Comment:
not related to your change, but `true &&` isn't it meaningless in that statement?

Reply:
nope. if CORVID_FORCE_HEADLESS is true, than the window will be hidden